assemblies/assembly-configuring-default-secret-pvc-mounts.adoc

```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="assembly-configuring-default-secret-pvc-mounts_{context}"]
+= Configuring default mounts for Secrets and PVCs
+
+You can configure Persistent Volume Claims (PVCs) and Secrets mount in your {product} deployment. Use annotations to define the custom mount paths and specify the containers to mount them to.
```

Comment:
Raised https://github.com/redhat-documentation/vale-at-red-hat/pull/932 to fix the following Vale alert:

[vale] reported by reviewdog 🐶
[RedHat.Spelling] Verify the word 'PVCs'. It is not in the American English spelling dictionary used by Vale.

Raw Output:
{"message": "[RedHat.Spelling] Verify the word 'PVCs'. It is not in the American English spelling dictionary used by Vale.", "location": {"path": "assemblies/assembly-configuring-default-secret-pvc-mounts.adoc", "range": {"start": {"line": 5, "column": 45}}}, "severity": "WARNING"}